src/agentplatform/adapter/runner.py

```
@@ -0,0 +1,141 @@
+    config 结构（render/targets.py 生成）：models.<alias> = {provider,
+    model, base_url(env:X), api_key(env:X)}（gateway 节合并 default）。
+    """
+    import yaml
+
+    cfg = yaml.safe_load((workspace / "config.yaml").read_text(encoding="utf-8"))
+    models = (cfg or {}).get("models") or {}
+    return {k: dict(v) for k, v in models.items() if isinstance(v, dict) and "model" in v}
+
+
+def resolve_flow_script(workspace: Path, team_id: str) -> Path:
+    """swarmflow/<team_id>.py（渲染 include_flows 产物）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)